examples/guestbook-operator/go.mod

```diff
@@ -10,4 +10,110 @@ require (
 	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20210922163802-cac4a6cf1977
 )
 
+require (
```

Comment:
/hold 

we ought to not have twice the required
we need to remove the indirect ones. 
to upgrade the go version we ought to not change the deps.